package.json

```diff
@@ -79,11 +97,14 @@
   },
   "devDependencies": {
     "@babel/preset-env": "^7.12.10",
+    "@frctl/fractal": "^1.5.4",
```

Comment:
Is it safe to remove this now?

Reply:
yeah, let's leap off that bridge.  We can probably remove the `njk` dependency too.